src/ai/backend/common/etcd.py

```diff
@@ -98,6 +98,8 @@ def _slash(v: str):
 P = ParamSpec("P")
 R = TypeVar("R")
 
+RecursiveMapping = Mapping[str, Union["RecursiveMapping", Optional[str]]]
```

Comment:
Let's rename this type name to somewhat related to `get_prefix()`. Vague names can create confusion to those who read the code.
```suggestion
GetPrefixValue = Mapping[str, Union["GetPrefixValue", Optional[str]]]
```

Reply:
Yes, it was too general.